src/components/ui/CustomCheckbox/CustomCheckbox.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box } from '@mui/material';
+import React, { useState } from 'react';
+
+interface ICustomCheckboxProps {
+  id: string;
+  children: React.ReactNode;
+  checked: boolean;
+  onChange: (checked: boolean) => void;
+}
+
+const styles = {
```

Comment:
its hardcode, you can use this https://mui.com/material-ui/customization/theme-components/